### ISSUE TITLE
[FR] ISO 12646 border config in GUI settings

### ISSUE THREAD

**Issue by Weissnix4711** [darktable-org/darktable#17278]
**Is your feature request related to a problem? Please describe.**

Cannot adjust ISO 12646 mode border settings without editing darktablerc. As per the [docs](https://docs.darktable.org/usermanual/4.8/en/module-reference/utility-modules/darkroom/color-assessment/):

> You can modify the distance from the edge of the center panel to the edge of the image by adjusting the darkroom/ui/iso12464_border entry in the $HOME/.config/darktable/darktablerc file. In the same file you can modify the size of the white border (in percent) by adjusting the darkroom/ui/iso12464_ratio entry.

This is not intuitive if using flatpak. It would be much better to just add `darkroom/ui/iso12464_border` and `darkroom/ui/iso12464_ratio` to the settings dialog.

**Describe the solution you'd like**

https://github.com/darktable-org/darktable/blob/768d772a49276e11babe68951b780742b793abc3/data/darktableconfig.xml.in#L2246

```
<dtconfig prefs="darkroom" section="general">
```

Correct me if I am wrong, but this should be all necessary to make the config point appear in the GUI settings?

Edit for anyone stumbling across this thread: Yes, the naming of the settings is incorrect. The standard is ISO 12646, but the settings are `iso12464_xxx`. Took me a second to figure out as well.

**Comment by Weissnix4711** [comment]
Fair enough, but I would argue for the border settings at least. Perhaps it could be kept seperate from the main settings page and instead have a right click menu like the overexposure indicator, softproof or gamut checking tool?

![image](https://github.com/user-attachments/assets/0d2fa571-e31e-40a2-b7a6-23f9cbf72c99)

I find the ISO 12646 tool can actually be rather useless on smaller monitors because the image itself is just too small.

**Comment by gi-man** [comment]
I agree with Ralf of too many settings can make the GUI too long/confusing, but I really like your proposal of using a right click menu.

**Comment by jenshannoschwalm** [comment]
> I find the ISO 12646 tool can actually be rather useless on smaller monitors because the image itself is just too small.

There **are** recommendations for "width of border" if using ISO 12646 assessment though ...

**Comment by da-phil** [comment]
@Weissnix4711 
I added the pop-up window you requested, as I also think it is a useful feature for 2 "hidden" configuration variables and for dialing in settings which are desired.
Please have a look and provide feedback: https://github.com/darktable-org/darktable/pull/18433

As written in the end of the PR description: I think there is something wrong with the border width config value, I could see any influence on the white border width...
